extensions/wikia/AdEngine/js/provider/gpt/adSizeFilter.js

```diff
@@ -19,41 +21,59 @@ define('ext.wikia.adEngine.provider.gpt.adSizeFilter', [
 	}
 
 	function getNewSizes(sizes, maxWidth, fallbackSizes) {
-		var goodSizes = [];
-
-		for (var i = 0; i < sizes.length; i += 1) {
-			if (sizes[i][0] <= maxWidth) {
-				goodSizes.push(sizes[i]);
-			}
-		}
+		var goodSizes = (sizes || []).filter(function(size) {
+				return size[0] <= maxWidth;
+			});
 
 		log(['getNewSizes', 'sizes: ', sizes, 'goodSizes: ', goodSizes], 'debug', logGroup);
 
 		return goodSizes.length ? goodSizes : fallbackSizes;
 	}
 
+	function removeSize(slotSizes, sizeToRemove) {
+		var adContext = win.Mercury ?
+				win.Mercury.Modules.Ads.getInstance().adsContext :
```

Comment:
Is it good idea to use Mercury modules even if it will be in app?

Reply:
nothing better comes to my mind in this case